xrootd/client/file_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package client // import "go-hep.org/x/hep/xrootd/client"
+
+import (
+	"context"
+	"testing"
+
+	"go-hep.org/x/hep/xrootd/xrdfs"
+)
+
+func testFile_Close(t *testing.T, addr string) {
+	client, err := NewClient(context.Background(), addr, "gopher")
+	if err != nil {
+		t.Fatalf("could not create client: %v", err)
+	}
+	defer client.Close()
+
+	fs := client.FS()
+
+	file, err := fs.Open(context.Background(), "/tmp/dir1/file1.txt", xrdfs.OpenModeOtherRead, xrdfs.OpenOptionsNone)
+	if err != nil {
+		t.Fatalf("invalid open call: %v", err)
+	}
+
+	err = file.Close(context.Background())
```

Comment:
could the other tests that `Open` a file be modified to add a `defer close` call?
so we don't leak file descriptors during those tests either.